### PR TITLE
NAD-1090 docs(numeric stepper): add design ref tab

### PR DESCRIPTION
**Pull request by kevinch**
#### What is the purpose of this pull request?
Add design ref tab.

#### What problem is this solving?
Lack of content.

#### How should this be manually tested?
See links below.

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Quality improvement (tests or refactors)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Trivial change (small fix or feature that doesn't impact functionalities)
- [ ] Requires change to documentation, which has been updated accordingly

### CODE REVIEW

File: packages/admin-ui-docs/docs/guidelines/components/numeric-stepper.mdx [hunk 1]
```diff
@@ -159,3 +185,18 @@ function Example() {
 | criticalText | `string`                    | Error message                            | 🚫       | -         |
 | helperText   | `string`                    | Helper text message                      | 🚫       | -         |
 | step         | `number`                    | Increment and decrement multiplier value | 🚫       | `1`       |
+
+  </Panel>
+  <Panel id="designReference">
+
+## Do’s
+
+1. Always use a default value. If you don't know which do use, use 1. This way the user is not forced to type something and then click the buttons.
```
Comment (anitavincent) [inline packages/admin-ui-docs/docs/guidelines/components/numeric-stepper.mdx:194]: which to*

Comment (davicostalf) [reply packages/admin-ui-docs/docs/guidelines/components/numeric-stepper.mdx:194]: 👆

Comment (kevinch) [reply packages/admin-ui-docs/docs/guidelines/components/numeric-stepper.mdx]: Updated.

**Review by davicostalf — APPROVED** [review]
LGTM

**Review by anitavincent — APPROVED** [review]

**Pull request merged**